common/flatpak-dir.c

```diff
@@ -7594,6 +7618,15 @@ flatpak_dir_install (FlatpakDir          *self,
         }
       else
         {
+          g_autofree gchar *bindfs_repo_path = NULL;
+          g_autoptr(GFile) bindfs_repo_file = NULL;
+          g_auto(GLnxLockFile) bindfs_repo_lock = { 0, };
+          g_autoptr(OstreeRepo) bindfs_repo = NULL;
+
+          flatpak_dir_system_helper_call_create_system_child_repo (self, ref, opt_commit,
+                                                                   installation ? installation : "",
+                                                                   &bindfs_repo_path, cancellable, error);
```

Comment:
Any error returned here is ignored. Then we unconditionally call `flatpak_dir_create_system_child_repo` below – but I think we should only do that if this call fails with `G_DBUS_ERROR_UNKNOWN_METHOD`.

Reply:
These bit is addressed in b55725c
Can you take a look at that first and reply?